nion/swift/model/DisplayItem.py

```diff
@@ -1996,11 +2002,13 @@ def append_display_data_channel(self, display_data_channel: DisplayDataChannel,
             self.__add_display_layer_auto(display_layer, display_data_channel)
 
     def __get_unique_display_layer_color(self) -> str:
+        # get a set of currently used colors w/o alpha, and then check if they're in a static color list
         existing_colors = {display_layer.fill_color for display_layer in self.display_layers}
-        for color in ('#1E90FF', "#F00", "#0F0", "#00F", "#FF0", "#0FF", "#F0F", "#888", "#800", "#080", "#008", "#CCC", "#880", "#088", "#808", "#964B00"):
-            if not color in existing_colors:
+        existing_colors = set(map(lambda c: "#" + c[3:] if isinstance(c, str) and len(c) == 9 else c, existing_colors))
```

Comment:
minor change here: can you using `DrawingContext.color_without_alpha` to convert the existing colors to hex values and then compare the hex values in all cases? That way it will handle strings like "red" and "rgb(255,0,0)" as the same. (on second look, I think you'll need to add a similar function in `DrawingContext` - something like `canonical_color_without_alpha` that always returns hex colors of length 9. That will be a separate PR.

Reply:
One additional change that would be nice - you can use the named colors instead of the hex colors in the DEFAULT_COLORS list. The last color is the list can use "sienna" as the named color.

Reply:
Also, the first color is "dodgerblue"